src/Glutinum.Converter/GlueAST.fs

```diff
@@ -340,3 +353,4 @@ type GlueType =
             match utilityType with
             | GlueUtilityType.Partial _
             | GlueUtilityType.Record _ -> "obj"
+        | MappedType _ -> "anonymous"
```

Comment:
This should be `obj`

If for some reason we encounter this case in the code generation `anonymous` will generate invalid F# code.

Reply:
Ok, that makes sense